features/publishable/publishable.feature

```diff
@@ -0,0 +1,97 @@
+    Given I add "Accept" header equal to "application/ld+json"
+    And I add "Content-Type" header equal to "application/ld+json"
+
+  # GET collection
+  @createSchema
+  Scenario: As a user with draft access, when I get a collection of published resources with draft resources available, it should include the draft resources instead of the published ones.
+
+  @createSchema
+  Scenario: As a user with draft access, when I get a collection of published resources with draft resources available, and published=true query filter, it should include the published resources only.
+
+  @createSchema
+  Scenario: As a user with no draft access, when I get a collection of published resources with draft resources available, it should include the published resources only.
+
+  @createSchema
+  Scenario: As a user with no draft access, when I get a collection of published resources with draft resources available, and published=true query filter, it should not include the draft resources.
+
+  # POST
+  @createSchema
+  Scenario: As a user with draft access, when I create a resource, I should have the draft resource returned.
+
+  @createSchema
+  Scenario: As a user with draft access, when I create a resource with an active publication date, I should have the published resource returned.
+
+  @createSchema
+  Scenario: As a user with draft access, when I create a resource with a future publication date, I should have the draft resource returned.
+
+  @createSchema
+  Scenario: As a user with no draft access, when I create a resource, I should have the published resource returned, and the publication date is automatically set.
+
+  # GET item
+  @createSchema
+  Scenario: As a user with draft access, when I get a published resource with a draft resource available, I should have the draft resource returned.
+  # todo Check cache-expiry header
+
+  @createSchema
+  Scenario: As a user with draft access, when I get a published resource with a draft resource available, and published=true query filter, I should have the published resource returned.
+  # todo Check cache-expiry header
+
+  @createSchema
+  Scenario: As a user with draft access, when I get a draft resource with published=true query filter, I should have a 404 error.
+  # todo Check cache-expiry header
+
+  @createSchema
+  Scenario: As any user, when I get a resource with a past publication date, and a draft resource available with an active publication date, the draft resource replaces the published one, and the old one is removed.
+  # todo Check cache-expiry header
+
+  @createSchema
+  Scenario: As a user with no draft access, when I get a published resource with a draft resource available, I should have the published resource returned.
+  # todo Check cache-expiry header
+
+  @createSchema
+  Scenario: As a user with no draft access, when I get a published resource with a draft resource available, and published=false query filter, I should have the published resource returned anyway.
```

Comment:
I'm not sure whether this is required, we may not be altering anything for 'published=false' for any user. We only adjust for published=true - OK to keep if it's just for completeness in case we make changes to reading this query-string in future

Reply:
`published=false` should not affect anything, but this scenario checks this security